scripts/daily_plot.py

```diff
@@ -102,7 +119,7 @@ def create_plot(df_plt_today, now, is_top=None):
 
     # Generate frequency plot
     plot = sns.countplot(y='Com_Name', hue='Com_Name', legend=False, data=df_plt_selection_today,
-                         palette=colors, order=freq_order, ax=axs[0])
+                         palette=colors, order=freq_order, ax=axs[0], edgecolor='lightgrey')
```

Comment:
the edge color is set for the default theme here too, I guess this is not the intent here?

Reply:
That was intentional. While it is more pronounced with the dark theme even with the green theme there are times the bar from the bar chart is white or almost white and blends into the chart background. Adding the border around the bar helps it stand out from the background.

<img width="322" alt="image" src="https://github.com/user-attachments/assets/dcc29cea-34cf-40a1-aaf6-901f8428c798">

Reply:
yes, it does look better with the edge